PermanentOrg/web-app
# Install and integrate MixPanel

Install the `mixpanel-browser` package and its corresponding type definitions, add an optional secret to take in a MixPanel project token, and identify users to MixPanel on login.

**Open Questions**
Do we want to have any additional configuration at all?

I decided to label users in testing environments with the environment they're on, since I'm not actually sure how MixPanel works and if there are other ways to filter out users from environments. Since I'm just passing `accountId`, there would be potential for overlap between environments otherwise. Would love some insight on this.

**Steps to Test**
1. Set MIXPANEL_TOKEN in your `.env` file to the project token.
2. Load up the web-app and verify that MixPanel events are being set. You should be able to see debug output in the developer console if you are in a non-production environment.
